pkg/blobinfocache/boltdb/boltdb.go

```diff
@@ -324,8 +336,52 @@ func (bdc *cache) appendReplacementCandidates(candidates []prioritize.CandidateW
 // If !canSubstitute, the returned candidates will match the submitted digest exactly; if canSubstitute,
 // data from previous RecordDigestUncompressedPair calls is used to also look up variants of the blob which have the same
 // uncompressed digest.
+//
+// If scope is nil, final result also contains blobs with lowest priority without locations that could be possibly be reused.
 func (bdc *cache) CandidateLocations2(transport types.ImageTransport, scope types.BICTransportScope, primaryDigest digest.Digest, canSubstitute bool) []blobinfocache.BICReplacementCandidate2 {
-	return bdc.candidateLocations(transport, scope, primaryDigest, canSubstitute, true)
+	resAllBlobs := []prioritize.CandidateWithTime{}
+	var uncompressedDigestValue digest.Digest // = ""
+	v2Output := true
```

Comment:
Renaming the caller-side argument without renaming the callee-side parameter, or changing what the function does, is rather confusing.

Or is that still WIP?

Reply:
Part of this is merged with candidateLocations.